chart/templates/injector_role.yaml

```diff
@@ -16,9 +16,14 @@ rules:
   - list
   - get
   - update
+  - watch
 - apiGroups:
   - ""
   resources:
   - services
+  - namespaces
   verbs:
+  - list
   - get
+  - update
```

Comment:
Hi @clairecng, @ptnapoleon 👋

I was going through the 6.0.0 release notes.
Was there any reason behind these additional permissions? Being a bit careful, especially with `update` permissions on services and namespaces.

Reply:
Ahh, hmm, @clairecng why did we need update for services actually?

Reply:
I noticed these permissions have now been removed. Thanks for that!